src/tink/http/Cookie.hx

```diff
@@ -0,0 +1,97 @@
+package tink.http;
+
+using DateTools;
+using StringTools;
+using Std;
+
+@:forward
+abstract Cookie(CookieObject) from CookieObject {
+	
+	public inline function new(name:String, value:String, ?expires:Date, ?domain:String, ?path:String, secure = false, httpOnly = false)
+		this = {
+			name: name,
+			value: value,
+			expires: expires,
+			domain: domain,
+			path: path,
+			secure: secure,
+			httpOnly: httpOnly,
+		}
+	
+	public static function parse(s:String):Array<Cookie> {
+		return [
+			for(pair in s.split('; ')) {
+				var v = pair.split('=');
+				{
+					name: v[0],
+					value: v[1],
+				}
+			}
+		];
+	}
+	
+	@:to
+	public function toString():String {
+		var buf = new StringBuf();
+		buf.add(this.name);
+		if(this.value != null) buf.add('=${this.value}');
+		if(this.expires != null) buf.add('; expires=${formatDate(this.expires)}');
+		if(this.domain != null) buf.add('; domain=${this.domain}');
+		if(this.path != null) buf.add('; path=${this.path}');
+		if(this.secure) buf.add('; secure');
+		if(this.httpOnly) buf.add('; httponly');
+		return buf.toString();
+	}
+	
+	function formatDate(date:Date) {
+		var timezoneOffset = 
```

Comment:
Oh, lucky you :)

Yeah, I think it's handled all the way down to the OS (or so it seems). And it's a PITA to deal with. I suppose if we could do all of this in UTC itself, it would be a lot easier.